### DISCUSSION TITLE
Are environment variables included when pushed to git?

### DISCUSSION THREAD

**Discussion by alaz-aura** [Kong/insomnia#7547] [Help] [answered]
hi all, when using the git option to sync collections, are the environments and all the key/value variables also pushed to git? Are they kept local? Curious how this works if anyone has any info/references to point to. Cheers!

**Comment by subnetmarco** [comment, marked as answer]
You can create two types of environments, public/shared and private. The private ones are not synched on Git or in the Cloud, the public/shared ones are.

![Screenshot 2024-06-18 at 10 04 21 PM](https://github.com/Kong/insomnia/assets/964813/0a783c4f-d900-4f45-890d-510e85e95bbb)

**Reply by alaz-aura** [reply]
Cheers @subnetmarco !
